listenbrainz/webserver/views/api.py

```diff
@@ -31,6 +31,26 @@
 
 DEFAULT_NUMBER_OF_PLAYLISTS_PER_CALL = 25
 
+SEARCH_USER_LIMIT = 10
+
+@api_bp.route('/playlist/search/users/', methods=['GET', 'OPTIONS'])
+@crossdomain
+@ratelimit()
+def search_user():
+    search_term = request.args.get("search_term")
+    user = validate_auth_header()
+    user_id = user.id
+    if search_term:
+        users = db_user.search(search_term, SEARCH_USER_LIMIT, user_id)
```

Comment:
Let us do this, as there can be need in the frontend/client to get more users than the default count value
```suggestion
        count = get_non_negative_param('count', SEARCH_USER_LIMIT)
        users = db_user.search(search_term, count, user_id)
```